tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -884,6 +888,13 @@ void test_zoned_time_formatter() {
     assert(format(STR("{:%g %G %U %V %W}"), zt) == STR("21 2021 16 16 16"));
 }
 
+template <typename CharT>
+void test_localized() {
+    auto loc = locale("ru-RU");
+    assert(format(loc, STR("{:%S}"), std::chrono::milliseconds(42)) == STR("00.042"));
+    assert(format(loc, STR("{:L%S}"), std::chrono::milliseconds(42)) == STR("00,042"));

```

Comment:
Nitpicks: This test is already using `std::chrono`. (In product code we must suffer through verbosity, but in test code we typically rely on using-directives to make the code more readable.)

I would additionally recommend the `42ms` user-defined literal, for consistency with the rest of the test, although it isn't important.

Reply:
Changed to `42ms`.